scripts/pipe_to_rshim.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+#
+# This script reads from a named pipe and writes to the rshim device.
+#
+# It is intended to be run on BMC to forward data from the host to the BMC RSHIM
+# device. 
```

Comment:
Have you considered auto-generating this script at runtime with a function in bfb-install? Or else we'll need to use a better name, and add it into .deb and .rpm package which seems a little conplicated.

Reply:
Done.